### ISSUE TITLE
Setting dragons targett location doesn't work

### ISSUE THREAD

**Issue by ItzMace** [CitizensDev/Citizens2#289]
Hey!

I have the problem, that my Enderdragon doesn't work like I want it to do!

I want my dragon to fly to a specific location and then to idle there (nothing else)!
But what I get is this: https://www.youtube.com/watch?v=tg9yEwuecd4

Here is some code trying to control the dragon:
dragon.faceLocation(targetLoc);
dragon.getNavigator().setTarget(targetLoc);

It seems to be the Minecraft AI, that lets the dragon go crazy.

What is my misstake?

Hope for help!

Best wishes,
ItzMace ;)

**Comment by Morphan1** [comment]
It almost looks like it's flying beyond the point and not being able to stay correctly. Try setting the speed to something lower... Something like
```
dragon.getNavigator().getLocalParameters().speedModifier(0.1f);
```

**Comment by ItzMace** [comment]
Hey! Thanks for your answer!
I now have this code:
army.faceLocation(balloonLoc);
army.getNavigator().setTarget(balloonLoc);
army.getNavigator().getLocalParameters().speedModifier(0.1f);

And this is what I get: https://www.youtube.com/watch?v=bul9H_JMZfs

I just want him to fly to a specific location and then to idle.

Hope you have another idea ;)

Best wishes,
ItzMace ;)

**Comment by fullwall** [comment]
Can you try setting the distance margin to be a bit higher?

**Comment by ItzMace** [comment]
Hey fullwall!
Thanks for your answer!

I set the dragons distance margin from 2.0 to 100.0 by:
dragon.getNavigator().getLocalParameters().distanceMargin(100.0);

This is nearly what I wanted.
But if the dragon reaches the location, it glitches a bit left.
I tried to stop him from moving with 
dragon.getNavigator().setPaused(true);
That works quite fine.

I'm now testing a bit round and writing if there are some problems left.

Best wishes,
ItzMace ;)
